clients/fides-js/src/components/Overlay.tsx

```diff
@@ -21,22 +24,43 @@ export interface OverlayProps {
   experience: PrivacyExperience;
   cookie: FidesCookie;
   fidesRegionString: string;
+  modalLinkEl?: HTMLElement | null;
 }
 
 const Overlay: FunctionComponent<OverlayProps> = ({
   experience,
   options,
   fidesRegionString,
   cookie,
+  modalLinkEl,
 }) => {
   const [modalIsOpen, setModalIsOpen] = useState(false);
+  const [bannerIsOpen, setBannerIsOpen] = useState(false);
 
   if (!experience.experience_config) {
+    debugLog(options.debug, "No experience config found");
     return null;
   }
 
   const privacyNotices = experience.privacy_notices ?? [];
 
+  if (modalLinkEl) {
+    debugLog(
+      options.debug,
+      "Modal link element found, updating it to show and trigger modal on click."
+    );
+    // Update modal link to trigger modal on click
+    const modalLink = modalLinkEl;
+    modalLink.onclick = () => {
+      setModalIsOpen(true);
+      setBannerIsOpen(false);
+    };
+    // Update to show the pre-existing modal link in the DOM
+    modalLink.style.display = "inline";
```

Comment:
nit: the CSS for the banner show/hide uses the classname `.fides-banner-hidden` instead of mutating styles directly.

It might be better to add/remove a class to the link element similarly:
```css
#fides-modal-link {
  display: none;
}

#fides-modal-link.fides-modal-link-shown {
  display: inline;
}
```